Lib/xml/etree/ElementTree.py

```diff
@@ -189,7 +189,7 @@ def makeelement(self, tag, attrib):
         """
         return self.__class__(tag, attrib)
 
-    def copy(self):
+    def __copy__(self):
```

Comment:
I think this breaks the public API.  I think we could do it in this way, keep the `copy()` method and alias it to `__copy__`.

Reply:
If you feel strongly about it, leave a comment on the bpo ticket. It may be worth having a discussion about what constitutes the "public" API in this case.